helm/plater/values.yaml

```diff
@@ -28,24 +28,33 @@ neo4j:
     failureThreshold: 3
     timeoutSeconds: 2
     periodSeconds: 10
-    tcpSocket:
-      port: 7687
+    exec:
+      command:
+        - bash
+        - -c
+        - IFS="/" read -a auth <<< $NEO4J_AUTH ; cypher-shell -u ${auth[0]} -p ${auth[1]} "match (c) return count(c)"
 
   livenessProbe:
     initialDelaySeconds: 60
     periodSeconds: 10
     failureThreshold: 3
     timeoutSeconds: 2
-    tcpSocket:
-      port: 7687
+    exec:
+      command:
+        - bash
+        - -c
+        - IFS="/" read -a auth <<< $NEO4J_AUTH ; cypher-shell -u ${auth[0]} -p ${auth[1]} "match (c) return count(C)"
```

Comment:
you have a mismatch of case between the (c) and count(C)

Reply:
ah thanks , had done that for the test